### DISCUSSION TITLE
Exceeding number of etiquetes sent

### DISCUSSION THREAD

**Discussion by mbaggnx** [porrey/Virtual-ZPL-Printer#25] [General]
Hello there

Just reached like the "maximum" amount of editions send
It is possible to make it "infinit" possiblity? or somehow to change the amount of receptions?

For example, if max is 50, you can change to 200 or 1000

Thanks, kind regards

<img width="519" alt="2023-02-09_15h25_00" src="https://user-images.githubusercontent.com/119416732/217841109-55ed906b-b564-4045-825a-35135ffcafaa.png">

**Comment by porrey** [comment]
This error is not from my application but comes from the Labelary web service.

**Comment by Bombera** [comment]
Hello Porrey 

we using your program and have the same error but we not print 50 labels we try print only 10pcs but recive this message. Please can you help me with this?
